roles/build-operand/tasks/main.yml

```diff
@@ -0,0 +1,52 @@
+---
+- name: Make sure imagestream name is set
+  when: not build_operand_imagestream_name
+  fail: msg="Failed because build_operand_imagestream_name has to be set"
+
+- name: Apply the namespace manifest
+  shell:
+    set -o pipefail;
+    cat "{{ build_operand_namespace_deployment }}" \
+    | sed 's|{{ '{{' }} build_operand_namespace {{ '}}' }}|{{ build_operand_namespace }}|' \
+    | oc apply -f-
+
+- name: Apply the imagestream manifest
+  shell:
+    set -o pipefail;
+    cat "{{ build_operand_imagestream }}" \
+    | sed 's|{{ '{{' }} build_operand_namespace {{ '}}' }}|{{ build_operand_namespace }}|' \
+    | sed 's|{{ '{{' }} build_operand_imagestream_name {{ '}}' }}|{{ build_operand_imagestream_name }}|' \
+    | oc apply -f-
+
+- name: Search if the CI image exists
+  command: oc get imagestreamtag -n "{{ build_operand_namespace }}" "{{ build_operand_imagestream_name }}:{{ build_operand_tag }}" -oname --ignore-not-found
```

Comment:
can you split in multiple lines, grouping together flags&values, eg
```
oc get imagestreamtag
   -n "{{ build_operand_namespace }}" 
   "{{ build_operand_imagestream_name }}:{{ build_operand_tag }}"
  -oname 
   --ignore-not-found
```
and I usually use `--ignore-not-found=true` to be more explicit, because all the `oc` commands don't have the same default value. Maybe it's redundant, idk ...